tools/assets_database/main.py

```diff
@@ -101,17 +100,9 @@ def main() -> None:
     msg_aggregator = MessagesAggregator()
     # We need a user db since the spam assets get synced during user unlock and
     # they touch both the global and the user DB at the same time
-    with TemporaryDirectory() as tmp_dir:
-        db = DBHandler(
```

Comment:
How come this was not found by any linter?

Reply:
not sure tbh